authors/apps/authentication/models.py

```diff
@@ -83,9 +83,13 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     # A timestamp reprensenting when this object was last updated.
     updated_at = models.DateTimeField(auto_now=True)
+    
+    is_valid =models.BooleanField(default=False)
 
     # More fields required by Django when specifying a custom user model.
+    email_verified = models.BooleanField(default=False)
```

Comment:
@Rhytah , any reason you adding these fields

Reply:
I added these fields for viewing the details of verification state in the database. In a scenario that a user claims to be verified and may be contrary.

Reply:
@Rhytah , concentrate on the scope of the story for now.
you can bring it up as an enhancement story later on

Reply:
@kimbugp in line with the story scope can I remove the fields?

Reply:
@kimbugp in line with the story scope can I remove the fields?

Reply:
@Rhytah , yes you can